content/articles/superhero-api/index.md

```diff
@@ -0,0 +1,471 @@
+                    <td>{{hero.biography.aliases}}</td>
+                </tr>
+                <tr>
+                    <td>Place of Birth</td>
+                    <td>{{hero.biography.placeOfBirth }}</td>
+                </tr>
+                <tr>
+                    <td>First Appearance</td>
+                    <td>{{hero.biography.firstAppearance }}</td>
+                </tr>
+                <tr>
+                    <td>Publisher</td>
+                    <td>{{hero.biography.publisher }}</td>
+                </tr>
+                <tr>
+                    <td>Alignment</td>
+                    <td>{{hero.biography.alignment }}</td>
+                </tr>
+            </tbody>
+        </table>
+    </section>
+</div>
+```
+
+4. Occupation Details
+
+```html
+  <div class="detail-item">
+    <span class="anchor" id="occupation-target"></span>
+    <h1>Occupation</h1>
+    <section id="occupation">
+        <table class="table table-bordered">
+            <thead>
+                <tr>
+                    <th>Metrics</th>
+                    <th>Measure</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Occupation</td>
+                    <td>{{hero.work.occupation}}</td>
+                </tr>
+                <tr>
+                    <td>Base</td>
+                    <td>{{hero.work.base}}</td>
+                </tr>
+            </tbody>
+        </table>
+    </section>
+</div>
+```
+
+5. Connection Details
+
+```html
+<div class="detail-item">
+    <span class="anchor" id="connections-target"></span>
+    <h1>Connections</h1>
+    <section id="connections">
+        <table class="table table-bordered">
+            <thead>
+                <tr>
+                    <th>Metrics</th>
+                    <th>Measure</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Group Affiliation</td>
+                    <td>{{hero.connections.groupAffiliation}}</td>
+                </tr>
+                <tr>
+                    <td>Relatives</td>
+                    <td>{{hero.connections.relatives }}</td>
+                </tr>
+            </tbody>
+            </table>
+    </section>
+</div>
+```
+![Single hero2](/engineering-education/superhero-api/single-hero2.png)
+
+### The static folder
+The static folder contains all the static files required for the project. These files include images, css, javascript, and other media files.
+
+Add the following snippets to the `settings.py` file to set up the static folder.
+
+```py
+#at the beginning of the file
+from pathlib import Path
+import environ
+import os
+
+
+# at the end of the file
+env = environ.Env()
+environ.Env.read_env()
+
+STATIC_URL = '/static/'
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'static'),
+)
+```
+
+You can  find the css and js file in [this link]
```

Comment:
@victorelvice 

Provide this link. I think you must have missed it.

Thanks!